### DISCUSSION TITLE
許多天前繳費后，一直沒有續期，請解決

### DISCUSSION THREAD

**Discussion by lushiyi23** [getlantern/lantern-forum-cn#963] [General]
10月2日續費，但至今10.14沒有完成續期。賬號為771673115@qq.com

**Comment by lushiyi23** [comment]
![dc81ecf5e72fbb9f1df9f4645951df63](https://github.com/user-attachments/assets/b782276b-5a1a-4a09-8b98-6e3324ee7fb1)
這是當時的記錄

**Comment by MaxMKMax** [comment]
@lushiyi23 好的. 已经帮您提交问题了.

**Reply by lushiyi23** [reply]
今天郵箱又收到一封郵件，讓我提交付款記錄，我試圖回復，但總是顯示被退回。

**Reply by MaxMKMax** [reply]
@lushiyi23 您好, 请问您的帐号状态怎么样? 可以用专业版吗? 那个邮件应该不重要的, 不需要理会. 谢谢!

**Reply by lushiyi23** [reply]
您好，目前仍然無法登錄。之前購買後沒有給我驗證碼，也沒有延長；現在授權登錄的郵件也發不到郵箱。

**Comment by lushiyi23** [comment]
目前仍然無法登錄，沒有激活碼，也收不到郵箱授權的郵件。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月19日(星期六) 凌晨0:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 許多天前繳費后，一直沒有續期，請解決 (Discussion #963)





 
@lushiyi23 您好, 请问您的帐号状态怎么样? 可以用专业版吗? 那个邮件应该不重要的, 不需要理会. 谢谢!

**Comment by MaxMKMax** [comment]
@lushiyi23 您好. 已经帮您提交问题上去了. 会尽快给您解决. 不好意思让您久等了. 谢谢您的耐心等待.
